src/System.Net.Security/src/System/Net/GlobalSSPI.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Net
+{
+    // Need a global so we can pass the interfaces as variables.
+    internal static class GlobalSSPI
```

Comment:
Design: Given the changes, this class doesn't make sense now and it should be removed.

`GlobalSSPI` was the factory class for the two Windows SSPIs. Since SSPIInterface (PAL) is now TLS-specific.

Suggested plan:
  1. Push GlobalSSPI and the two SSPIInterface implementations based on Schannel down in the Windows-specific implementation. Regardless of changes at this layer, this abstraction still captures the essence of how Schannel SSPIs work.
  2. Redesign a cleaner PAL that would satisfy the needs for TLS
  3. (Future) design a new PAL for Nego
  4. Do a commonality/variability analysis between the two TLS and Nego PALs. If it makes sense anymore, bring the factory back.